src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -60,23 +60,17 @@ private void MainForm_Load(object sender, EventArgs e)
 		{	
 			UpdateWindowTitle();
 			
-			ToolStripItem[] CreateWindowSizeFactorSubmenus()

```

Comment:
Unrelated removal of this local method?

Reply:
I had to change this method to insert the items at the top of the menu (above the separator and new item added by this PR), and since there's no `InsertRange` method or similar the simplest way was inserting the items directly and dropping the whole array -> `AddRange` step.

Since theres nothing to return, the local method would just be

```csharp
void CreateWindowSizeFactorSubmenus() { ... }
CreateWindowSizeFactorSubmenus();
```

Reply:
Oh right. I suggest you add those new items programmatically rather than in the Designer. Alternatively, just add a "scoping block" around this bit and re-indent it, that will make the diff readable.